src/pages/home/ReportScreen.js

```diff
@@ -73,6 +75,11 @@ class ReportScreen extends React.Component {
         this.loadingTimerId = setTimeout(() => this.setState({isLoading: false}), 300);
     }
 
+    storeCurrentlyViewedReport() {
```

Comment:
All our components methods must have a method doc even if they don't take params or return anything.

Reply:
> Avoid docs that don't add any additional information.
> Method descriptions are optional and should be added when it's not obvious what the purpose of the method is.
https://github.com/Expensify/Expensify.cash/blob/master/STYLE.md#jsdocs 🤓 

This is a class method and I'll just be adding this as a description: "Persists the currently viewed report id". I could extend it to "...with Onyx" but this is just an implementation detail 

Anyway this is not the first time, someone would bring up func/method documentation. In fact it is always the case that I should add documentation so you might consider updating the guide - I would have just always added documentation if I hadn't read the above in the style guide.

Reply:
Thanks for calling that out. I will update it to be more clear.